### PR TITLE
Update and pin Snapcast and librespot versions

### PR DESCRIPTION
**Pull request by endotronic**
Thanks for putting this container together! I was having trouble getting it to work, and I found that the snapserver CLI has changed in ways that broke this container. I've pinned all the versions to avoid this problem and reworked the initialization so that it should "just work." I removed the mechanism to specify Spotify username and password as I find it works just fine without them.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,30 +1,37 @@
...
 
 RUN apt-get update \
- && apt-get -y install curl libportaudio2 libvorbis0a libavahi-client3 libflac8 libvorbisenc2 libvorbisfile3 \
+ && apt-get -y install curl libportaudio2 libvorbis0a libavahi-client3 libflac8 libvorbisenc2 libvorbisfile3 libopus0 \
  && apt-get clean && rm -fR /var/lib/apt/lists
 
 ARG ARCH=amd64
-ARG SNAPCAST_VERSION=0.11.1
+ARG SNAPCAST_VERSION=0.19.0
+ARG LIBRESPOT_VERSION=0.1.1
```
Comment (djmaze) [inline Dockerfile:24]: This is a duplicate from above.

**Review by djmaze — COMMENTED** [review]
I don't get it. How can this possibly work without a username and password? Then there is no way to make a connection to your account. Does that mean anyone (in your wifi) can control the playback?

I have to take a deeper look at the rest of the changes and test them, thanks.

File: Dockerfile (hunk 1 above)
Comment (endotronic) [reply Dockerfile:24]: It must be duplicated as the second part is a second build. Unfortunate, and maybe there is a better way.

**Comment by endotronic** [comment]
Yeah, anyone on my local network can control it with their Spotify account, which is actually what I want. If you don't want that, perhaps we should make an option to specify username and password.

**Comment by djmaze** [comment]
That is interesting. I didn't know that was even possible. 

We should probably make this an option though.

File: init.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+snapserver -d --stream.sampleformat=44100:16:2
+librespot -n $DEVICE_NAME -b 320 --backend pipe > /tmp/snapfifo
```
Comment (djmaze) [inline init.sh:3]: So librespot must really be started separately now?

**Comment by djmaze** [comment]
Thanks @endotronic! I took your changes and made a few adjustments.

**Comment by endotronic** [comment]
Awesome, I'm glad!

**Pull request merged**